src/AccessControl.js

```diff
@@ -0,0 +1,187 @@
+import { ethers } from 'ethers';
+import { useCallback, useEffect, useState } from 'react';
+import { useSynthetix } from './useSynthetix';
+
+function AccessControl() {
+  const [synthetix, updateSynthetix] = useSynthetix();
+  const { contract, permissions, walletAddress } = synthetix;
+  const [userApproveWallet, setUserApproveWallet] = useState('');
+  const [userApproveWalletError, setUserApproveWalletError] = useState('');
+  const [userRevokeWallet, setUserRevokeWallet] = useState('');
+  const [userRevokeWalletError, setUserRevokeWalletError] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+  const [loadingText, setLoadingText] = useState('');
+
+  // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
```

Comment:
don't skip on exhaustive deps

Reply:
removed this case, thanks for your advice and approach!